x/vesting/keeper/integration_test.go

```diff
@@ -537,25 +871,87 @@ func delegate(clawbackAccount *types.ClawbackVestingAccount, amount int64) error
 	return err
 }
 
-func performEthTx(clawbackAccount *types.ClawbackVestingAccount) error {
-	addr, err := sdk.AccAddressFromBech32(clawbackAccount.Address)
-	s.Require().NoError(err)
+func createEthTx(privKey *ethsecp256k1.PrivKey, from sdk.AccAddress, dest sdk.AccAddress, amount *big.Int, nonceIncrement int) *evmtypes.MsgEthereumTx {
```

Comment:
ditto, seems like this is dup code

Reply:
I'm guessing it would make sense to include this in the generalized testing utils. 
Can't directly move this currently because the function is using several methods and fields from the suite struct.

I would suggest leaving this like it is and then moving on to work on the testing refactor?

Reply:
let's move it to testutils and make it public then